packages/openapi2-parser/lib/parser.js

```diff
@@ -325,6 +326,18 @@ class Parser {
     });
   }
 
+  // Add the link to the format version specs
+  addFormatLink() {
+    const { Link } = this.namespace.elements;
+    const link = new Link();
+
+    link.title = 'OpenAPI 2';
+    link.relation = 'via';
+    link.href = 'https://spec.openapis.org/oas/v2.0';
+
+    return this.api.links.push(link);
```

Comment:
The relation link should go into the parse result (`this.result`) directly instead of the API Category (`this.api`).

Reply:
I didn't get a chance to elaborate on the explanation so just adding that now.

The reason behind it, is that the api category represents the API itself, whereas the parse result represents the result of parsing the source document. Thus sementically, the parse result came "via" the source format, but does not represent information about the API itself (only how it was described).